source/MissionAction.cpp

```diff
@@ -33,6 +33,16 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 using namespace std;
 
 namespace {
+	void DoGift(PlayerInfo &player, const Ship *model, const string &name, UI *ui)
+	{
+		if(model->ModelName().empty())
+			return;
+		
+		player.BuyShip(model, name, true);
+		if(ui)
+			ui->Push(new Dialog("The " + name + " was added to your fleet!"));
```

Comment:
Perhaps `“A <model>, the <name>, was added to your fleet.”`?

Reply:
I think `The <model> <noun> "<name>" was added to your fleet.` or `You received the <model> <noun> "<name>."` would work.

To match outfit messaging, it could be `A/An <noun>, the <model> "<name>," was added to your fleet.`

Reply:
Based on community feedback on Discord as well, I went with `A/An <noun>, the <model>"<name>," was added to the fleet.`